portal/server/controllers/start/before-you-start.controller.js

```diff
@@ -0,0 +1,29 @@
+import { errorHref, generateErrorSummary } from '../../helpers';
+import beforeYouStartValidation from '../../validation/before-you-start';
+
+export const getBeforeYouStartPage = async (req, res) => {
```

Comment:
Docstring please.

Reply:
sorry -- These are just me pulling out controller functions into controllers -- If this is the method i'm going to go with, there'll be a huge effort in doing this, so I'm not looking to touch the actual content if thats okay?